src/index.js

```diff
@@ -12,7 +12,13 @@ program.on('--help', () => {
 
   $ ${
     pkg.name
-  } -o davfoundation -t "abcdef" -c 10 -e exclude.json -w output.json
+  } -o davfoundation -t "abcdef" -c 400 -e exclude.json -w output.json
+  $ ${
+    pkg.name
+  } -o davfoundation [-t "abcdef"] -e exclude.json -w output.json
```

Comment:
Do we need `[]` here... ? can we remove it ?

Reply:
I left it there to refer to it as being optional. 

https://stackoverflow.com/questions/21503865/how-to-denote-that-a-command-line-argument-is-optional-when-printing-usage

Reply:
If you changed your mind and you would rather keep the token for the second example, I'll change it right away!

Reply:
Hey @mrcolo since we are using this as an example, I would like to show practical use cases. We have already explained that few of the flags are optional in the help text. Hope its clear 🙂

Reply:
Alright! It should be fine now. Let me know.